homeassistant/components/folder_watcher/event.py

```diff
@@ -0,0 +1,101 @@
+"""Support for Folder watcher event entities."""
+
+from __future__ import annotations
+
+from typing import Any
+
+from watchdog.events import (
+    EVENT_TYPE_CLOSED,
+    EVENT_TYPE_CREATED,
+    EVENT_TYPE_DELETED,
+    EVENT_TYPE_MODIFIED,
+    EVENT_TYPE_MOVED,
+)
+
+from homeassistant.components.event import EventEntity, EventEntityDescription
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.helpers import device_registry as dr
+from homeassistant.helpers.dispatcher import async_dispatcher_connect
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .const import DOMAIN
+
+EVENT_DESCRIPTIONS = (
+    EventEntityDescription(
+        key=EVENT_TYPE_CLOSED,
+        translation_key=EVENT_TYPE_CLOSED,
+    ),
+    EventEntityDescription(
+        key=EVENT_TYPE_CREATED,
+        translation_key=EVENT_TYPE_CREATED,
+    ),
+    EventEntityDescription(
+        key=EVENT_TYPE_DELETED,
+        translation_key=EVENT_TYPE_DELETED,
+    ),
+    EventEntityDescription(
+        key=EVENT_TYPE_MODIFIED,
+        translation_key=EVENT_TYPE_MODIFIED,
+    ),
+    EventEntityDescription(
+        key=EVENT_TYPE_MOVED,
+        translation_key=EVENT_TYPE_MOVED,
+    ),
+)
```

Comment:
This seems odd, I would expect a single event entity for a folder?

Reply:
That's right. Not sure why I split it up but it's fixed now